### ISSUE TITLE
Add ability to see size of editor buffer (file not saved to disk)

### ISSUE THREAD

**Issue by magiruuvelvet** [mkxml/vscode-filesize#38]
When editing a buffer which wasn't saved on disk yet, the extension doesn't show anything. Would it be possible to show the size of the editor buffer when no file exists on disk (yet)?

**Comment by PSAGChristian** [comment]
Yes, sometimes it would be really helpful to be able to see the size of a file that is not yet saved on the hard disk (e.g. new file from clipboard).
